### PR TITLE
Alt Wysiwyg Implementation

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#16133

*Description of changes:*
* Builds a new wysiwyg editor using TipTap Editor and various extensions. 
* The purpose of this update editor, is it has a vue 3 supported instance of the editor, so alleviate the scope during vue upgrade process. 
* Includes some design features outlined in design: https://projects.invisionapp.com/share/W312QYCNE56Z#/screens/467365020
* The above includes: Enhanced/organized toolkit, headers, clear functionality and formatting.

*Notes:*
- This commit incudes a new feature flag to toggle between the old and new wysiwyg editor

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by cameron-eyds** [comment]
Currently expanding on unit testing for when we remove the old editor 👍

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1395-bstohhlo.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1395-bstohhlo.web.app

**Review by doug-lovett — APPROVED** [review]
Very nice.

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1395-bstohhlo.web.app

**Review by dimak1 — APPROVED** [review]
Nice and clean work!

File: ppr-ui/src/components/collateral/generalCollateral/GenColAmend.vue [hunk 1]
```diff
@@ -38,14 +47,22 @@
...
         <v-col class="pr-4">
+          <WysiwygEditor
+            v-if="isTiptapEnabled"
+            placeHolderText="Enter the General Collateral to be added to this registration"
+            :editorContent="addDesc"
+            @emitEditorContent="addDesc = $event"
+          />
+
           <tiptap-vuetify
+            v-if="false"
```
Comment (orelbn) [inline ppr-ui/src/components/collateral/generalCollateral/GenColAmend.vue]: Is this meant to be here?

**Review by orelbn — APPROVED** [review]
Looks great!

File: ppr-ui/src/components/collateral/generalCollateral/GenColAmend.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/collateral/generalCollateral/GenColAmend.vue]: No haha
thank you

Comment (cameron-eyds) [reply ppr-ui/src/components/collateral/generalCollateral/GenColAmend.vue]: Fixed ~in local commit~, good eye 👍

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1395-bstohhlo.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1395-bstohhlo.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1395-bstohhlo.web.app

**Pull request merged**